### DISCUSSION TITLE
IBEX on the cover of the Journal of Cell Science!

### DISCUSSION THREAD

**Discussion by radtkea** [IBEXImagingCommunity/ibex_imaging_knowledge_base#246] [Announcements]
We are so very proud of Aanandita Kothurkar, Ryan MacDonald, Colin Chu, and colleagues! Their beautiful and impactful work is featured on the [December 2024](https://journals.biologists.com/jcs/issue/137/23) cover of the Journal of Cell Science. Great work Aanandita on the striking cover image of the killifish retina!

In addition, their work has an accompanying [research highlight](https://journals.biologists.com/jcs/article/137/23/e137_e2301/363410/Shedding-light-on-retinal-cells) and [video](https://www.youtube.com/watch?v=N20u1F09gFU). Read the full article [here](https://journals.biologists.com/jcs/article/137/23/jcs263407/363401/Iterative-Bleaching-Extends-Multiplexity).
![joces_137_23 coverfig](https://github.com/user-attachments/assets/3f2767e0-c38c-40a0-a7ee-6e1a8e09f872)

**Comment by DasEFX130** [comment]
Love the image! Exciting to see.
